.github/workflows/refresh-wordpress-major-and-beta.yml

```diff
@@ -39,22 +39,25 @@ jobs:
                   curl -fsSL https://bun.sh/install | bash
             - uses: ./.github/actions/prepare-playground
             - name: 'Recompile WordPress'
+              id: build
               shell: bash
               env:
                   FORCE_REBUILD: ${{ github.event.inputs.force_rebuild }}
               run: PATH="${PATH}:${HOME}/.bun/bin" npx nx bundle-wordpress:major-and-beta playground-wordpress-builds
             - name: Check for uncommitted changes
+              # Prevent commit if the build failed to avoid polluting the commit history
+              if: steps.build.outcome == 'success'
```

Comment:
You are right, I kept the logic in place as I kept changing the PR, but haven't considered that it's not even needed. 

```suggestion
```